### DISCUSSION TITLE
Welcome to ispc Discussions!

### DISCUSSION THREAD

**Discussion by dbabokin** [ispc/ispc#1944] [General]
I'm turning on discussions for ISPC repo to see how it plays out.

Feel free to ask a question, throw in an idea, or share your experience using ISPC!

**Comment by Zerophase** [comment]
I just know Unreal uses a lot of ISPC, while compiling the main engine project. Get a lot of cache misses from ccache, on the editor. Everything else, without ISPC, is pretty much a 100% hit. It would be nice if ccache improved their ISPC support. Would definitely be hugely beneficial for anyone working with Unreal. There's a ccache patch for Unreal that would be perfect for testing something like this.

**Reply by dbabokin** [reply]
I've submitted an issue here: https://github.com/ccache/ccache/issues/847. Feel free to comment on the issue or contribute a patch.
